src/main/kotlin/com/dansplugins/fiefs/command/fiefs/create/FiefsCreateCommand.kt

```diff
@@ -0,0 +1,56 @@
+package com.dansplugins.fiefs.command.fiefs.create
+
+import com.dansplugins.fiefs.Fiefs
+import org.bukkit.ChatColor
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.command.TabCompleter
+import org.bukkit.entity.Player
+
+class FiefsCreateCommand(private val plugin: Fiefs) : CommandExecutor, TabCompleter {
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+        if (sender !is Player) {
+            sender.sendMessage("This command can only be run by a player.")
```

Comment:
Colours are consistent on other error messages

Reply:
Thanks for catching this! Fixed it!